src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/UpdateSentenceDateService.kt

```diff
@@ -166,12 +165,7 @@ class UpdateSentenceDateService(
     )
   }
 
-  private fun isNotApprovedForHdc(updatedLicenceEntity: Licence): Boolean {
-    val hdcStatus = prisonApiClient.getHdcStatus(updatedLicenceEntity.bookingId!!)
-      .defaultIfEmpty(PrisonerHdcStatus(passed = false, approvalStatus = "UNKNOWN"))
-      .block()!!
```

Comment:
pushed mono handling into client to simplify and tidy up tests.. also added helper functions to HDCStatus so we don't have to string compare all over